core/kazoo_speech/src/asr/kazoo_asr_google.erl

```diff
@@ -83,3 +105,36 @@ handle_response({'ok', _Code, _Hdrs, Content2}) ->
     lager:debug("asr of media failed with code ~p", [_Code]),
     lager:debug("resp: ~s", [Content2]),
     {'error', 'asr_provider_failure', kz_json:decode(Content2)}.
+
+%%------------------------------------------------------------------------------
+%% @doc Convert audio file/content-type if initial format not supported
+%% @end
+%%------------------------------------------------------------------------------
+-spec maybe_convert_content(binary(), kz_term:ne_binary()) -> conversion_return().
+maybe_convert_content(Content, ContentType) ->
```

Comment:
Is enough recognition quality for audio converted from mpd3 to wav?

Reply:
In my tests with MP3's recorded at 48khz it was good enough and the transcriptions were much better than iSpeech.